experimental/AlgebraicStatistics/src/CI.jl

```diff
@@ -31,27 +29,32 @@ to ensure consistent comparison and hashing.
 ## Examples
 
 ```jldoctest
-julia> ci_stmt(["A"], ["B"], ["X"])
-[A _||_ B | X]
+julia> ci_stmt([1], [2], [3,4])
+[1 _||_ 2 | {3, 4}]
 
-julia> ci_stmt(["1"], ["2", "3"], ["4", "5"])
+julia> ci_stmt([1], [2, 3], [4, 5])
 [1 _||_ {2, 3} | {4, 5}]
 ```
 """
-function ci_stmt(I::Vector{<:VarName}, J::Vector{<:VarName}, K::Vector{<:VarName}; symmetric=true, semigraphoid=true)
+function ci_stmt(I::Vector{Int}, J::Vector{Int}, K::Vector{Int}; symmetric=true, disjoint=true)
   if length(intersect(I, J)) > 0
-    error("Functional dependence statements are not yet implemented")
+    error("I and J may not overlap")
   end
+  I = sort(unique(I))
+  J = sort(unique(J))
   if symmetric && I > J
     I, J = J, I
   end
-  if semigraphoid
+  if disjoint
     I = setdiff(I, K)
     J = setdiff(J, K)
   end
-  CIStmt(sort(unique(I)), sort(unique(J)), sort(unique(K)))
+  CIStmt(sort(I), sort(J), sort(unique(K)))
```

Comment:
```suggestion
  CIStmt(I, J, sort(unique(K)))
```
These two `sort` should not be neessary, `I` and `J` are sorted and `setdiff` should preserve the order.